frontend/javascripts/dashboard/advanced_dataset/create_explorative_modal.tsx

```diff
@@ -65,18 +65,19 @@ export function NewVolumeLayerSelection({
         value={selectedSegmentationLayerIndex}
         disabled={disableLayerSelection ?? false}
       >
-        <Radio key={-1} value={-1}>
-          Create empty layer
-        </Radio>
         {segmentationLayers.map((segmentationLayer, index) => (
           <Radio
             key={segmentationLayer.name}
             value={index}
             disabled={!doesSupportVolumeWithFallback(dataset, segmentationLayer)}
+            style={{ fontStyle: "italic" }}
```

Comment:
I don't really understand why this is italic now? Making `Create empty layer` italic would make some sense to me, but turning all items italic except for the "empty layer" seems counter-intuitive to me (highlighting everything but one). maybe remove this line again?

Reply:
I wanted to highlight that the selection features a layer name and not a "technical term". In other words, the selectable options are named by a user and have no technical relevance.

In this screenshot I found it really odd that according to the wording my volume annotation is based an "a segmentation". `Segmentation` the name of a layer and not a technical property. This would have been clearer if the layer was called `synapse` or `cell borders` or something.
<img width="377" alt="image" src="https://user-images.githubusercontent.com/1105056/196430828-03e4379f-4a50-4e64-81bb-06a4b9cafabc.png">

Maybe we can make this more clear in a different way than italics.

Reply:
ah, I see. what about putting the layer names in quotation marks to make it clear that these are layer names? or put " layer" behind each name?